### PR TITLE
add desc to xeno doors

### LINKED ISSUE #2466: resin doors need a description

### PR DESCRIPTION
**Pull request by Hmeister-real**
## About the PR
 fixes #2466

## Why / Balance
issue

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Hmeister 
- tweak: Minor xeno description changes

### CODE REVIEW

File: Resources/Prototypes/_CM14/Entities/Structures/Xeno/xeno_doors.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
   id: DoorXenoResin # TODO CM14 make them only buildable next to other resin walls
   parent: CMBaseXenoStructure
   name: resin door
+  description: A resin door that only sisters may pass.
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_CM14/Entities/Structures/Xeno/xeno_doors.yml:5]: These descriptions are only supposed to be seen by xenos in cm13, I'll add a component to make this possible.
This is what a marine sees when examining a resin door:
image

**Review by DrSmugleaf — DISMISSED** [review]

File: Resources/Prototypes/_CM14/Entities/Structures/Xeno/xeno_doors.yml (hunk 1 above)
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Xeno/xeno_doors.yml:5]:
```suggestion
  description: That's a resin door.
```

**Review by QuietlyWhisper — APPROVED** [review]
Thanks

File: Resources/Prototypes/_CM14/Entities/Structures/Xeno/xeno_doors.yml [hunk 2]
```diff
@@ -84,6 +85,7 @@
   id: DoorXenoResinThick
   parent: DoorXenoResin
   name: thick resin door
+  description: A thick resin door, which is more durable, that only sisters may pass.
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Xeno/xeno_doors.yml:88]:
```suggestion
  description: A thick resin door, which is more durable.
```

**Pull request merged**